### ISSUE TITLE
Cvt pathway should be an autophagy

### ISSUE THREAD

**Issue by ukemi** [geneontology/go-ontology#13886]
After community feedback, we have decided to make the Cvt pathway a type of autophagy. Since currently autophagy is defined as being a catabolic pathway, we will change the name of the current term to catabolic autophagy and create a new autophagy grouping class. Catabolic autophagy and the Cvt pathway will then become children of the new class.

**Comment by ValWood** [comment]
Is that for sure? I relly thought  this pathway was for the delivery of *resident* hydrolases to the vacuole (i.e just getting things where they belong, not for catabolism.

Yes, it is "autophagy -related"  (uses much of the same machinery), and is used as a model for autophagy, but to call it a type of autophagy just seems wrong...

**Comment by ukemi** [comment]
@RLovering @Pauldenny @paolaroncaglia @marcfeuermann

**Comment by paolaroncaglia** [comment]
@ValWood 
Exactly. The new, generic 'autophagy' term will _not_ be linked to catabolism, neither will its child Cvt. Only the children of 'catabolic autophagy' (i.e. the current autophagy terms) will be linked to catabolism.
When we started work on the autophagy branch, we, too, didn't want to make Cvt a type of autophagy (and it won't be in the classical sense, even after David's edits, because it won't be a catabolic autophagic process). But the reviews we got back from a very autophagy-specific journal (and the feedback we specifically sought with the Editor) underline that to the autophagy community it is really important to have a link between Cvt and autophagy 'at large'. So we thought of various options, and resolved that David's suggestion above keeps things biologically correct and does not require co-annotation to catabolic processes. We'll also mark the generic 'autophagy' non-catabolic parent not for manual annotation. All details will become clear when David commits his changes, but if you have any serious concern, we'll post a sneak preview of some sort.
As always, thanks for your precious feedback :-)

**Comment by ValWood** [comment]
Maybe the journal editors don't understand GO and that the link between CVT and autophagy should be the shared genes annotated to the 2 processes ;)
Although this doesn't bother me much because as yet no annotations to CVT , I would be more worried if autophagy were not a catabolic process.
But no worries.

**Comment by ValWood** [comment]
Actually, now I re-read I do think it odd not to have all autophagy linked to catabolism. Maybe the correct questions were not asked,  or explanations given, to the advisors?  Sometimes it is difficult  to serve user expectations if the user/advisor  does not have full understanding  of GO and you might need to explain  the effects in more detail  (here, for example  there is possibly a misunderstanding between commonality in  processes and  annotations). This is  a good example of  introducing a high level  biologically non-sensical split  "autophagy/ catabolic autophagy" in the ontology to  serve a specific use-case that will cause problems later. I was discussing such  examples with @pgaudet  earlier.

Why did the journal editors want the connection? The CVT pathway and autophagy could have a common parent (relating to the common part of the process if that is biologically meaningful), but autophagy is a catabolic process.

**Comment by paolaroncaglia** [comment]
@ValWood 
We may need to move some of this discussion to a private channel. It might be more appropriate to treat details of reviewers' comments and positions as confidential while the paper is still in review. 
@ukemi would you agree?

**Comment by ValWood** [comment]
Don't worry, I spoke with David. We will just make sure we block the parent term for annotation and use the catabolic terms. Others can make their own adjustments as necessary.... to me it still  seems a strange un-GO like work-around  to change the definition of autophagy to be none- degradative  to accomodate a related process but  no worries, if users think that makes sense.

**Comment by ValWood** [comment]
Published today:

  | Galluzzi  L, Baehrecke EH, Ballabio A, Boya P, Bravo-San Pedro JM, Cecconi F,  Choi AM, Chu CT, Codogno P, Colombo MI, Cuervo AM, Debnath J, Deretic V,  Dikic I, Eskelinen EL, Fimia GM, Fulda S, Gewirtz DA, Green DR, Hansen  M, Harper JW, Jäättelä M, Johansen T, Juhasz G, Kimmelman AC, Kraft C,  Ktistakis NT, Kumar S, Levine B, Lopez-Otin C, Madeo F, Martens S,  Martinez J, Melendez A, Mizushima N, Münz C, Murphy LO, Penninger JM,  Piacentini M, Reggiori F, Rubinsztein DC, Ryan KM, Santambrogio L,  Scorrano L, Simon AK, Simon HU, Simonsen A, Tavernarakis N, Tooze SA,  Yoshimori T, Yuan J, Yue Z, Zhong Q, Kroemer G. | 
  | Molecular definitions of autophagy and related processes.
  | EMBO J. 2017 Jun 8;. [Epub ahead of print]
  | PMID: 28596378

Over the past two decades, the molecular machinery that underlies autophagic responses has been characterized with ever increasing precision in multiple model organisms. Moreover, it has become clear that autophagy and autophagy-related processes have profound implications for human pathophysiology. However, considerable confusion persists about the use of appropriate terms to indicate specific types of autophagy and some components of the autophagy machinery, which may have detrimental effects on the expansion of the field. Driven by the overt recognition of such a potential obstacle, a panel of leading experts in the field attempts here to define several autophagy-related terms based on specific biochemical features. The ultimate objective of this collaborative exchange is to formulate recommendations that facilitate the dissemination of knowledge within and outside the field of autophagy research.

see
We agree on two main features that characterize bona fide, functional autophagic responses, irrespective of type: (i) they involve cytoplasmic material; and (ii) they culminate with (and strictly depend on) lysosomal degradation.

and
Cytoplasm-to-vacuole targeting pathway
The cytoplasm-to-vacuole targeting (Cvt) pathway delivers hydro- lases including aminopeptidase 1 (Ape1), Ape4, and alpha-manno- sidase (Ams1) to the yeast vacuole (Umekawa & Klionsky, 2012). The molecular machineries for the Cvt pathway and macroau- tophagy share a large number of components, including several Atg proteins (Scott et al, 1996, 2000, 2001). Moreover, Ape1, Ape4, and Ams1 are imported into the vacuole as large oligomers, being reminiscent of the substrates of aggrephagy (Bertipaglia et al, 2016). The Cvt pathway, however, contributes to the preser- vation of normal enzymatic activity within the vacuole, especially in vegetative conditions, de facto mediating biosynthetic, rather than catabolic, functions (Umekawa & Klionsky, 2012). Thus, the Cvt pathway does not represent an instance of autophagy strictly speaking.


You should reconsider the new arrangment. it seems very wrong....

**Comment by ukemi** [comment]
It seems that this area is still in flux. Note the 'strictly speaking' qualifier above. To be conservative and I hopefully to satisfy everyone I propose the following:

1.  keep the current structure
2. renaming the new term to autophagy-like process retaining its definition and adding the new paper as another reference?   Currently looks like this--http://amigo.geneontology.org/amigo/term/GO:0061919
3. Changing the name of the original term back to simply autophagy and keeping everything else as is?  Currently looks like this--http://amigo.geneontology.org/amigo/term/GO:0006914

**Comment by RLovering** [comment]
Hi David

I think this is the best idea, what do you think @ValWood ?

Ruth

**Comment by krchristie** [comment]
Hi David,

I like your proposal a lot, though I have one minor issue.

It seems a little odd to me to classify 'autophagy' as an 'autophagy-like process', since is IS autophagy rather than merely like it. 

Perhaps we could come up with an alternate name for the grouping term. The name 'autophagy-related process' would suffer from the same issue as your current name, but perhaps 'autophagic machinery dependent process' would be better?

-Karen

**Comment by ValWood** [comment]
I don't know. This paper really does seem quite adamant. Even in their "key recommendations section" they say :
"Cytoplasm-to-vacuole targeting (Cvt), LC3-associated phagocytosis (LAP), crinophagy, and instances of protein secretion that depend on components of the macroautophagy apparatus are not bona fide autophagic responses."
so it seems odd to make term called "autophagy-like process" term to get it under here autophagy.
I can't think of a precedent....

I prefer Karen's suggestion "autophagic machinery dependent process". 
At least anybody looking for autophagy would not be confused that "autophagy" was classed as an "autophagy-like process", and it is much clearer what the processes  do have in common.

**Comment by ukemi** [comment]
I could go with 'autophagic machinery dependent process', but I think it is a bit awkward. If we make the autophagy-like/machinery dependent process term, we aren't classifying it as a bona fide type of autophagy. We are classifying it as a process that is similar to autophagy. I think this is consistent with everyone's view or we wouldn't be having this discussion. There is precedence for the 'like' terms. See 'ubiquitin-like protein ligase activity' and its children.

**Comment by ValWood** [comment]
ubiquitin-like protein ligase activity is a bit different, since it's a MF and there are lots of modifiers which are "ubiquitin-like" but we don't have a collective noun for them.

I was trying to recall an example where processes with unrelated outcomes use the same molecular machinery....there must be some but I couldn't  think of any....

**Comment by tberardini** [comment]
Also, paper from Feb. 2017: (from the 'CVT is autophagy' camp)

http://www.sciencedirect.com/science/article/pii/S0022283617300177
“Cytoplasm-to-vacuole targeting (Cvt) is a selective autophagy pathway that specifically transports vacuolar hydrolases into the vacuole in budding yeast cells and has been extensively studied as a model of selective autophagy. “
17 February 2017

**Comment by ukemi** [comment]
It is also supported in PMID:22966490. This is the paper that was the origin of the definition of the generic autophagy term. It is authored by a large number of members of the autophagy community.

**Comment by ValWood** [comment]
All of the above references call CVT "autphagy related/like" (used as a model for autophagy, not autophagy). None of them say it IS autophagy.
My problem was in having CVT a a descendant of "autophagy", and introducing "catabolic autophagy" when autophagy is *always catabolic*. That should be part of the autophagy definition.

As pointed out   above I agree  it would  also seem odd to have "autophagy" classed as "autophagy-related/like"

The clunky "autophagic machinery dependent process" (or along these lines) is  still probably the best name because  the machinery used is really the commonality. It is awkward, but it is only a grouping term, it won't be used for  direct annotation....

**Comment by RLovering** [comment]
Hi All
sorry to be annoying but we need to get this resolved this week, so that we can submit our paper next week hopefully.

The situation is that there are 2 camps here. Both of which have the backing of substantial numbers of well respected autophagy researchers. From our point of view the editor of the journal 'autophagy' is in the camp where Cvt is considered an 'autophagic process'. It is not unusual to find that biologists disagree on aspects of biology and usually we wait for this area to be resolved. In this situation we were unaware of the opposing camp and followed the reviewers comments (not the editors comments) which had questioned why Cvt wasn't listed as a child of autophagy.

To be clear, I think we are all agreed that the term catabolic autophagy (nee autophagy) will be renamed back to autophagy. The question just remains about whether there should be a grouping term for (catabolic) autophagy, Cvt and exophagy.

Note that the name of the ATG# genes is: autophagy related #. 

Options appear to be:
autophagy-related process
autophagy-like process
autophagic machinery dependent process
autophagy component dependent process

I will contact the authors of this other paper and see what response we get there. 

Ruth

**Comment by RLovering** [comment]
Hi All

please see response from author of the recent autophagy paper Val pointed us to. I think this is a valid comment and maybe would support our discussion with other camp.

Date: Monday, 24 July 2017 at 16:32
To: "Lovering, Ruth" <r.lovering@ucl.ac.uk>, "deadoc@vodafone.it" <deadoc@vodafone.it>
Cc: David Hill <David.Hill@jax.org>
Subject: RE: Gene Ontology description of autophagy
 
Dear Ruth,
 
many thanks for your message, I am sorry to hear that the publication of our paper was problematic on your end.
 
It is obviously difficult to adopt a term that groups autophagy, Cvt and ATG-dependent secretion, as functionally they little (if any) similarities, but in some cases happen to rely on shared molecular machinery.
 
From my point-of-view (which does not necessarily reflect that of my co-authors), this is like attempting to group CASP3-dependent cell death and CASP3-dependent neuronal differentiation because they both happen to involve CASP3. What term would you use in this case? Cell-death related process? Misleading. Differentiation-related process? Cell death machinery dependent process? Equally misleading.
 
We discovered autophagy and Cvt “first” so they are better characterized as compared to ATG-dependent secretion, but we have no clues on potential hierarchical links between these processes, if any. Most likely, they are instances of parallel and efficient evolution, taking advantage of existing proteins for the establishment of new functions.
 
More pragmatically, I would avoid the term autophagy from the parent term, perhaps replacing it with “Trafficking”, “Sorting” or something alike. Matter of fact, the endocytic, exocytic and VPS pathways seem to me as hierarchically equivalent to autophagy, CVT, and ATG-dependent secretion (implying they should go under the same parent term and its definition should be changed)
 
Hope my thoughts (which, again, do not necessarily reflect those of my co-authors) help
 
Please feel free to get back in touch as needed
 
All the best
 
Lorenzo
 
 
Lorenzo Galluzzi, Ph.D.
Weill Cornell Medical College - Assistant Professor
Université Paris Descartes - Honorary Associate Professor

OncoImmunology - Editor in Chief
Molecular and Cellular Oncology - Editor in Chief
International Review of Cell and Molecular Biology - Editor in Chief
Oncotarget "Autophagy and Cell Death" - Section Chief Editor

Stich Radiation Oncology
525 East 68th Street, Box # 169
New York, NY 10065
USA

**Comment by ukemi** [comment]
OK. It seems like there are certainly two camps. Here is what I propose, hopefully satisfying everyone to some extent:

1) Rename the current autophagy term (GO:0061919) to 'process utilizing autophagic mechanism' keeping the definition as is: A cellular process involving delivery of a portion of the cytoplasm to lysosomes or to the plant or fungal vacuole that does not involve direct transport through the endocytic or vacuolar protein sorting (Vps) pathways. This process typically leads to degradation of the cargo; however, it can also be used to deliver resident proteins, such as in the cytoplasm-to-vacuole targeting (Cvt) pathway.

This term will still be marked 'do not annotate'.

2) Revert the name of 'catabolic autophagy (GO:0006914) back to autophagy and change the def from: The autophagy process in which cells digest parts of their own cytoplasm; allows for both recycling of macromolecular constituents under conditions of cellular stress and remodeling the intracellular structure for cell differentiation.
to: The cellular process in which cells digest parts of their own cytoplasm; allows for both recycling of macromolecular constituents under conditions of cellular stress and remodeling the intracellular structure for cell differentiation.

I think this organization will allow for future refinementsas the filed matures and the mechanisms that undelie all of these processes are better understood. Based on the comments and definition, there is also the possibility of making the parent term have a has_part relationship to 'establishment of protein localization to organelle'. 

Please comment on this by July 26th because I would like to get these changes finalized.

**Comment by RLovering** [comment]
Thanks David

I think this is the best way forwards

Ruth

**Comment by ValWood** [comment]
I think any of your solutions are better than the original.
"catabolic autophagy" has filtered through to PomBase and you have to admit that out of context it looks a tad strange!
http://pombase2.bioinformatics.nz/term/GO%3A0006914

I like  "process utilizing autophagic mechanism" it very clearly indicates  why the parent is there.
v

**Comment by RLovering** [comment]
Thanks for confirming Val

good to have agreement all round

**Comment by marcfeuermann** [comment]
Hi,
Sorry for this late reply. David, I agree with your latest changes.
CVT uses autophagy components to target proteins such as aminopeptidase I to be active in the vacuole, but not for their degradation. So I would not consider it as autophagy but a "transport to the vacuole using autophagy machinery" with parent "transport". Catabolic autophagy looks really weird.
Best regards,
Marc

**Comment by ukemi** [comment]
I have made all of the changes except the has-part relationship to the localization term. Based on the commentary, there is a has-part relationship to some type of localization, but because some selective autophagy processes don't target proteins, we can't say it is protein localization. The only term left is very general and isn't really worth making a has-part relation to. If we expand the localization part of the ontology in the future, we can consider this again.

**Comment by ukemi** [comment]
I just merged these changes into the live ontology. I also ended up tweaking some definitions and removing some synonyms that were no longer valid based on the new ontology structure. Hopefully I dotted all the 'i's and crossed all the 't's. Let me know if you spot anything that is missing or inconsistent. I will leave this open, but I am creating a figure based on the new structure.

**Comment by paolaroncaglia** [comment]
Hi @ukemi
Minor comment about GO:0061912 selective autophagy ("The macroautophagy process in which specific structures are targeted by the autophagy process."). Should this be marked 'not for manual annotation', since we'd expect curators to know what structure is targeted and therefore choose the most appropriate child (or suggest a new one)? I might have missed discussion on this while I was on leave, sorry if so.
Thanks,
Paola

**Comment by paolaroncaglia** [comment]
Hi again @ukemi
Wrt spotting missing or inconsistent items among the most recent autophagy edits, I'll place this here even if it doesn't relate to Cvt. I just noticed that the recently renamed GO:0044804 autophagy of nucleus (was 'nucleophagy') does not retain nucleophagy as a synonym, not even related. Same for the recently renamed GO:0030242 autophagy of peroxisome (was 'pexophagy', even though 'pexophagy' now exists to replace 'macropexophagy') and GO:0000422 autophagy of mitochondrion (was 'mitophagy', similar to the previous example). Wouldn't it be better to keep the old names as related synonyms, even if they exist as primary names elsewhere? I recall that for legacy purposes some groups prefer to keep at least a trace of previous primary names, and there are other cases in GO of related synonyms being the same as primary names for other terms. As long as a synonym is not exact, that kind of duplication should not cause technical issues.
Thanks,
Paola

**Comment by ukemi** [comment]
Added synonyms. The only outstanding item is to add the do not annotate tag to the selective autophagy term. I will add that to the list as part of @BarbaraCzub 's training.

**Comment by paolaroncaglia** [comment]
Thanks @ukemi!
